### DISCUSSION TITLE
Some URLs don't parse and show anymore.

### DISCUSSION THREAD

**Discussion by antdude** [sopel-irc/sopel#2183] [Q&A] [answered]
In my friend's Fedora v33 via SSH2 shell account, I no longer see my Sopel v7.1.4 | Python: 3.9.7 parsing and showing some URLs (e.g., http://aqfl.net and http://youtube.com) in IRC's channels. http://github.com/sopel-irc/sopel worked though.

$ cat /home/ant/.sopel/logs/default.error.log
[2021-09-15 14:04:34,468] sopel.bot            ERROR    - Cannot register URL callback: Bugzilla URL callback requires "bugzilla.domains" to be configured; check your config file.

Is this related? If so, then how do I fix it? Thank you for reading and hopefully answering soon. :)

**Comment by dgw** [comment, marked as answer]
Multiple past discussions and issues with you have expressed that you cannot expect YouTube links to work without [the youtube plugin](https://pypi.org/project/sopel-modules.youtube/). That hasn't changed.

I can't reproduce any issue fetching the title for http://aqfl.net/ on our bot in #sopel on Libera. Something must be getting blocked, maybe because your friend is using a shell service. Anything with fairly open access tends to get IP-blocked by providers to combat abuse.

Error is unrelated. `sopel-plugins disable bugzilla` if it bugs you.

**Comment by antdude** [comment]
Thanks. https://pypi.org/project/sopel-modules.youtube/ worked for YouTube issue. Hmm, I will have to look at the other issues.
